scripts/build.sh

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env bash
 
-if [ -z "$1" ]; then 
+if [ -z "$1" ]; then
     BUILD_TYPE=Release
-else 
+else
     BUILD_TYPE="$1"
 fi
 
-cd $(dirname "$0")
-mkdir -p build
-cd build
-cmake .. -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
+cd $(dirname "$0")/.. &&
```

Comment:
correct, but either way the .git dir is going to be present, will it not? Unless...is the .git dir taken out of a zip download? (But even then...they're just getting the binary, no?)